files/tinywww-debug.yml

```diff
@@ -18,7 +18,7 @@ spec:
         app: tinywww
     spec:
       containers:
-      - image: embano1/tinywww:latest
+      - image: projects.registry.vmware.com/veba/tinywww:latest
```

Comment:
Assuming you pushed these images based on the original sources?

Reply:
That's correct